code/modules/power/cell.dm

```diff
@@ -280,6 +280,7 @@
 	maxcharge = 50000
 	rating = 12
 	ratingdesc = FALSE
+	self_recharge = 1

```

Comment:
Well it still corrects the code to fit the description, which should correct any other silly interactions with emp.